ras_party/controllers/account_controller.py

```diff
@@ -190,6 +203,37 @@ def change_respondent_enrolment_status(payload, session):
                         desc='No active enrolments remaining for case')
 
 
+@with_db_session
+def disable_all_respondent_enrolments(respondent_email, session):
+    """Disables all enrolments for a respondent ,  returns the count of the removed enrolments"""
+
+    obfuscated_email = obfuscate_email(respondent_email)
+
+    logger.info('Disabling all enrolments for respondent', email=obfuscated_email)
+
+    removed_enrolments_count = 0
+
+    # raises errors if none or multiple, unusual import to avoid circular references
+    respondent = __import__('ras_party').controllers.respondent_controller.get_single_respondent_by_email(
+        respondent_email, session)
+
+    enrolments = query_all_non_disabled_enrolments_respondent(respondent.id, session)
+
+    for enrolment in enrolments:
+        _change_respondent_enrolment_status(respondent=respondent,
+                                            survey_id=enrolment.survey_id,
+                                            business_id=enrolment.business_id,
+                                            status='DISABLED',
+                                            session=session)
+        removed_enrolments_count += 1
+
+    # If no enrolments then logged count is zero
```

Comment:
No need for this comment, as you've defined the enrolment count slightly higher in the code